lib/tasks/decidim_kids.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+namespace :decidim_kids do
+  desc "Create default minors account static page"
+  task create_default_page: :environment do
+    Decidim::Organization.find_each do |organization|
+      Decidim::System::CreateDefaultPages.call(organization)
```

Comment:
This creates a default minor page for all organizations, as the page created is a `DEFAULT` one, it means it cannot be removed by the admins. As this module needs to be enabled in the system configuration, the organizations without it enable will still have this minor page created without being able to remove it.

Reply:
Task removed as the default page for minors is created when creating and updating the organization and the kids configuration is enabled